source/includes/mongodb-compatibility-table-node.rst

```diff
@@ -13,7 +13,7 @@
      - MongoDB 4.0
      - MongoDB 3.6
 
-   * - 6.10 and 6.11
+   * - 6.10 and 6.12
```

Comment:
Should this be "to" instead of "and"? Otherwise it seems it's omitting 6.11
```suggestion
   * - 6.10 to 6.12
```

Reply:
great catch